documentation/source/faq/main.rst

```diff
@@ -0,0 +1,212 @@
+- If you have found a way to set it up for other Linux distributions, please `open a pull request <https://github.com/inexorgame/vulkan-renderer/pulls>`__ and let us know!
+- We do not support macOS or iOS because it would require us to use `MoltenVK <https://github.com/KhronosGroup/MoltenVK>`__ to get Vulkan running on Mac OS. Additionally, this would require some changes in the engines as not all of Inexor's dependencies are available on macOS or iOS.
+- We also do not support Android because this would require some changes in the engines as not all of Inexor's dependencies are available on Android.
+
+----
+
+What is the current state of the project?
+-----------------------------------------
+
+We are still in very early development, but this project can already offer:
+
+- A modern C++17 codebase with a setup for CMake and conan package manager.
+- Stable builds for Windows and Linux using `Continuous Integration (CI) <https://en.wikipedia.org/wiki/Continuous_integration>`__.
+- A `rendergraph <https://de.slideshare.net/DICEStudio/framegraph-extensible-rendering-architecture-in-frostbite>`__ in early development.
+- `ImGui <https://github.com/ocornut/imgui>`__ integration using separate renderpasses.
+- `RAII <https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rr-raii>`__ wrappers for various Vulkan resources.
+- Extensive logging with `spdlog <https://github.com/gabime/spdlog>`_.
+- `Vulkan Memory Allocator <https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator>`__ for graphics memory management.
+- `VMA memory replays <https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator#binaries>`__ for debugging are already working.
+- Full `RenderDoc <https://renderdoc.org/>`__ integration with `internal resource naming <https://www.saschawillems.de/blog/2016/05/28/tutorial-on-using-vulkans-vk_ext_debug_marker-with-renderdoc/>`__.
+
+----
+
+How is Inexor organized?
+------------------------
+
+- Inexor has no central authority.
+- It's a headless collective which makes collective decisions through discussions.
+- It's welcoming new contributors.
+
+----
+
+How to contact us?
+------------------
+
+Please visit `inexor.org <https://inexor.org>`__ and join our `Discord <https://discord.com/invite/acUW8k7>`__ server.
+
+----
+
+How to build?
+-------------
+
+If you have any trouble building please `open a ticket <https://github.com/inexorgame/vulkan-renderer/issues>`__ or `join our Discord <https://discord.com/invite/acUW8k7>`__.
+
+`How to build vulkan-renderer? <https://inexor-vulkan-renderer.readthedocs.io/en/latest/development/building.html>`__
+
+Where to find Inexor's documentation?
+-------------------------------------
+
+- Read our docs `here <https://inexor-vulkan-renderer.readthedocs.io/en/latest/>`__.
+
+----
+
+What is Vulkan API?
+-------------------
+
+.. image:: ../links/VulkanAPI.png
```

Comment:
Here we have the same problem as pointed out earlier.

Reply:
```suggestion
.. image:: /links/VulkanAPI.png
```

Reply:
done.